lib/query.js

```diff
@@ -276,7 +276,7 @@ Query.prototype.toConstructor = function toConstructor() {
  * @api public
  */
 
-Query.prototype.clone = function clone() {
+Query.prototype.clone = function() {
```

Comment:
consider re-adding a name, will be better for debugging

Reply:
The problem there is naming this `clone` conflicts with the `clone` import. So not ideal, but not a blocker.